File: Project2BackEnd/src/main/java/com/Project2BackEnd/service/JobService.java
package com.Project2BackEnd.service;

import java.util.List;

import com.Project2BackEnd.model.Job;

public interface JobService {
	
	List<Job> getAllJobs();
	void postJob (Job job);
	void deleteJob( int jobId);
	Job getJobById(int jobId);
}

File: Project2FrontEnd/WebContent/job/jobservice.js
app.factory('JobService',function($http){
	var jobService={};
	var BASE_URL="http://localhost:8085/Project2BackEnd"
		jobService.saveJob=function(job){
		return $http.post(BASE_URL + "/postJob",job);
		
	}
	
	jobService.getAllJobs=function(){
		return $http.get(BASE_URL + "/getAllJobs");
	}
	
	jobService.deleteJob=function()
	{
		return $http.delete(BASE_URL + "/job/{jobId}");
	}
	return jobService; 
})